test/tst.common.js

```diff
@@ -160,6 +160,20 @@ function runTests(cons, label)
 	    '    at runTests (dummy filename)',
 	    '    at Object.<anonymous> (dummy filename)'
 	].join('\n') + '\n' + nodestack);
+
```

Comment:
Is there a reason to put this under the 'custom "name"' test, instead of under the simple one (e.g., at line 82, where we check "err.stack" for the simple case)?